Dockerfile

```diff
@@ -1,8 +1,9 @@
-FROM alpine:3.12
+FROM alpine:3.13
 
 RUN apk --no-cache add ca-certificates
 
 COPY cr /usr/local/bin/cr
 
-# Ensure that the binary is available on path and is executable
-RUN cr --help
```

Comment:
I think it's ok to keep this just to ensure it's ok

Reply:
ok i can remove the commit.

From my perspective it is more clear what the docker image does with the chmod command. It increase the trust.

Reply:
@davidkarlsen done